pkg/cloudprovider/plugins.go

```diff
@@ -132,3 +141,38 @@ func InitCloudProvider(name string, configFilePath string) (Interface, error) {
 
 	return cloud, nil
 }
+
+// RegisterKMSService registers a cloudprovider.KMSService by name.  This
```

Comment:
Add to the comment what the name is used for. (I assume it must match the value in the encryption provider config file?)